tox.ini

```diff
@@ -0,0 +1,16 @@
+[tox]
+envlist = py26, py27, py33, py34, py35, pypy
+
+[testenv]
+passenv = HYPOTHESIS_STORAGE_DIRECTORY
+
+deps =
+  nose-cov
+  hypothesis
+
+commands =
+  nosetests -v --logging-level=DEBUG --with-cov --cov chardet --cov-config .coveragerc
```

Comment:
[Travis](https://travis-ci.org/chardet/chardet/jobs/84036215) says otherwise:

```
$ python --version
Python 2.7.8 (10f1b29a2bd2, Feb 02 2015, 21:22:43)
[PyPy 2.5.0 with GCC 4.6.3]
$ pip --version
pip 6.0.7 from /home/travis/virtualenv/pypy-2.5.0/site-packages (python 2.7)
install
1.65s$ pip install tox
You are using pip version 6.0.7, however version 7.1.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
Collecting tox
  Downloading tox-2.1.1-py2.py3-none-any.whl
Collecting virtualenv>=1.11.2 (from tox)
  Downloading virtualenv-13.1.2-py2.py3-none-any.whl (1.7MB)
    100% |################################| 1.7MB 284kB/s 
Requirement already satisfied (use --upgrade to upgrade): py>=1.4.17 in /home/travis/virtualenv/pypy-2.5.0/site-packages (from tox)
Collecting pluggy<0.4.0,>=0.3.0 (from tox)
  Downloading pluggy-0.3.1-py2.py3-none-any.whl
Installing collected packages: pluggy, virtualenv, tox
Successfully installed pluggy-0.3.1 tox-2.1.1 virtualenv-13.1.2
1129.72s$ tox -e py
GLOB sdist-make: /home/travis/build/chardet/chardet/setup.py
py create: /home/travis/build/chardet/chardet/.tox/py
py installdeps: nose-cov, hypothesis
py inst: /home/travis/build/chardet/chardet/.tox/dist/chardet-2.3.0.zip
py installed: cffi==0.8.6,chardet==2.3.0,cov-core==1.15.0,coverage==4.0,greenlet==0.4.5,hypothesis==1.11.4,nose==1.3.7,nose-cov==1.6,readline==6.2.4.1,wheel==0.24.0
py runtests: PYTHONHASHSEED='2593478130'
py runtests: commands[0] | nosetests -v --logging-level=DEBUG --with-cov --cov chardet --cov-config .coveragerc
```